content/push/setup.textile

```diff
@@ -0,0 +1,451 @@
+---
+title: Setup and device management
```

Comment:
I wonder if "Device setup and activation" would be a better title and also reflected in the nav. WDYT?

Reply:
Or even just "Device setup"?